src/main/java/at/hannibal2/skyhanni/data/GardenCropMilestones.kt

```diff
@@ -47,6 +57,41 @@ object GardenCropMilestones {
         GardenCropMilestonesCommunityFix.openInventory(event.inventoryItems)
     }
 
+    fun onOverflowLevelUp(crop: CropType, oldLevel: Int, newLevel: Int) {
+        val customGoalLevel = ProfileStorageData.profileSpecific?.garden?.customGoalMilestone?.get(crop) ?: 0
+        val goalReached = newLevel == customGoalLevel
+
+        val rewards = buildList {
```

Comment:
I still think we should be combining this logic with the SkillProgress but maybe this can be done in a future pr

Reply:
resolved by adding todos